setup.py

```diff
@@ -8,21 +8,20 @@
 
 setup(
     name='mysql-autoxtrabackup',
-    version='1.5.4',
-    packages=['general_conf', 'backup_prepare', 'partial_recovery', 'master_backup_script', 'prepare_env_test_mode'],
+    version='1.5.5',
+    packages=['general_conf', 'backup_prepare', 'partial_recovery', 'master_backup_script', 'prepare_env_test_mode', 'process_runner'],
     package_data={
         'prepare_env_test_mode': ['*.sh', '*.sql']
     },
     py_modules=['autoxtrabackup'],
     url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup',
-    download_url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup/archive/v1.5.4.zip',
+    download_url='https://github.com/ShahriyarR/MySQL-AutoXtraBackup/archive/v1.5.5.zip',
     license='MIT',
     author='Shahriyar Rzayev',
     author_email='rzayev.shahriyar@yandex.com',
     description='Commandline tool written in Python 3 for using Percona XtraBackup',
     install_requires=[
         'click>=3.3',
-        'mysql-connector-python==2.1.4',
```

Comment:
sure??

Reply:
Master said yes :)
https://github.com/ShahriyarR/MySQL-AutoXtraBackup/blob/51afdd437fd67bede9719d46c5d5b0c94a6304dc/setup.py#L26

Reply:
but where is the mysql connector library then? (sorry i'm blind xD)

Reply:
It turned out there is no need for this library :D ahaahah )

Reply:
o_0 ok then lol